run-build.sh

```diff
@@ -116,7 +120,10 @@ while [[ $# > 0 ]]; do
             echo "  --nobuild                           Skip building, showing the command that would be used to build"
             echo "  --docker <IMAGENAME>                Build in Docker using the Dockerfile located in scripts/docker/IMAGENAME"
             echo "  --linux-portable                    Builds the Linux portable .NET Tools instead of a distro-specific version."
-            echo "  --all-linux-installers              Builds and publishes all the Linux distros' native installers."
+            echo "  --all-linux-installers              Builds and publishes all the Linux distros' native installers; outer call"
```

Comment:
Can't we just use the all-linux-installers parameters for this? Also, why isn't passing the target directly not working anymore?

Reply:
'Why isn't passing the target directly not working anymore?' - because defining 2 targets on the first 'dotnet msbuild' call in 'run-build.sh' is where the break is occurring. 

'Can't we just use the all-linux-installers parameters for this?' - There is an 'outer' or entry point call to 'build.sh' with a "BuildAndPublishAllLinuxDistrosNativeInstallers" target in addition to an 'inner' or a recursive call to 'build.sh' [see: SandBoxAndPackage.proj ] with a "GenerateInstallersAndCopyOutOfSandBox" target - they appear to need separate treatment.